_pages/en/building-enabling-directorate.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: default
+title: Building Nested Enabling Teams
+ref: mandate
+lang: en
+status: posted
+categories: Work In Progress
+permalink: /building-enabling-directorate.html
+---
+
+## {{ page.title }}
+
+### Problem Statement
+
+In [Team Topologies](https://itrevolution.com/book/team-topologies/) they write that organizations should organize themselves by nesting the four team definitions.
+
+- Value Stream Aligned Teams
+- Platform Teams
+- Enabling Teams
+- Complex Subsystem Teams
+
+In our particular case, to use it as an example, the IT Strategy team is an enabling team. We are under a director who considers the directorate to be an enabling team.  Therefore the IT Strategy team is an enabling team within an enabling team, and has other sibling teams that are also enabling teams. That being the case, how would one synchronize activities of the entire directorate? Say the director was to offer their services to a team elsewhere in the organization. How best to manage and coordinate the work between the series of sub-enabling teams? That is the question this document is intending to answer.
```

Comment:
Would be worthwhile to have a one-two liner on what is an enabling team. e.g.
"those which are more expertise-focused. They facilitate other teams to do their work and to gain capabilities and be able to do all the things [like] testing, monitoring, deployment, etc., without requiring experts in each of those areas"

Reply:
Agreed, or a definitions section might be useful. We need to keep in mind most people reading our content will not have a similar background. Just linking books will not cut it 😀 (A lot of people don't have time/interest to read work related books)